### DISCUSSION TITLE
Mobile support

### DISCUSSION THREAD

**Discussion by moraaar** [NVIDIA-Omniverse/PhysX#19] [Q&A] [answered]
Hello,

I was wondering if there is (or will be) mobile support (iOS and Android) on PhysX 5.

Thanks

**Comment by amoravanszky** [comment, marked as answer]
We don't plan it from our side but if the community would like to make a port and maintain it, we can talk about how to best help and enable that.

**Comment by moraaar** [comment]
I've created a PR adding Android support to PhysX 5.1: https://github.com/NVIDIA-Omniverse/PhysX/pull/40

Cheers,
Aaron

**Comment by moraaar** [comment]
PR adding iOS support to PhysX 5.1: https://github.com/NVIDIA-Omniverse/PhysX/pull/49

Any questions about on Android or iOS support PRs don't hesitate to ask.

Thanks,
Aaron
